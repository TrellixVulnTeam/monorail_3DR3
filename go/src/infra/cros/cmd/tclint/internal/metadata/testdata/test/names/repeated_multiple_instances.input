{
    "remoteTestDrivers": [
        {
            "name": "remoteTestDrivers/tauto",
            "tests": [
                {
                    "name": "remoteTestDrivers/tauto/tests/foo"
                },
                {
                    "name": "remoteTestDrivers/tauto/tests/foo"
                },
                {
                    "name": "remoteTestDrivers/tauto/tests/baz"
                },
                {
                    "name": "remoteTestDrivers/tauto/tests/baz"
                }
            ]
        }
    ]
}