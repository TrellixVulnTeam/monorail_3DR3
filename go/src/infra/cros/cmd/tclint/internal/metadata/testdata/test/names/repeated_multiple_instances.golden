[
"## Errors ##",
"RemoteTestDriver 'remoteTestDrivers/tauto': Test names must be unique, found repeated name(s): 'remoteTestDrivers/tauto/tests/baz','remoteTestDrivers/tauto/tests/foo'"
]